download.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: download
+title: "Download - Jamulus"
+lang: "en"
+permalink: /download
+---
+    <div class="fx-row fx-row-between-xs container">
```

Comment:
I'm still not 100% convinced if we really want to maintain one page for every language with that much HTML. Yes, the index.html page looks quite similar.

Reply:
With luck it'll hardly change once it's settled down.